aptos-move/framework/aptos-stdlib/sources/cryptography/bn254_algebra.move

```diff
@@ -0,0 +1,915 @@
+/// This module defines marker types, constants and test cases for working with BN254 curves using the generic API defined in `algebra.move`.
+/// BN254 was sampled as part of the [\[BCTV14\]](https://eprint.iacr.org/2013/879.pdf) paper .
+/// The name denotes that it is a Barreto--Naehrig curve of embedding degree 12,
+/// defined over a 254-bit (prime) field. The scalar field is highly 2-adic.
+///
+/// This curve is also implemented in [libff](https://github.com/scipr-lab/libff/tree/master/libff/algebra/curves/alt_bn128) under the name `bn128`.
+/// It is the same as the `bn254` curve used in Ethereum (eg: [go-ethereum](https://github.com/ethereum/go-ethereum/tree/master/crypto/bn254/cloudflare)).
+///
+/// #CAUTION
+/// **This curve does not satisfy the 128-bit security level anymore.**
```

Comment:
Can you add a `(see "Updating Key Size Estimations for Pairings"; by Barbulescu, Razvan and Duquesne, Sylvain; in Journal of Cryptology; 2019; https://doi.org/10.1007/s00145-018-9280-5)` note here please?

Reply:
For now, I just added the complement after the line. If it's not the case, please let me know. Many thanks.